src/pages/bots/[id]/index.tsx

```diff
@@ -18,14 +16,32 @@ const BotDetail: NextPage = () => {
   const { id } = useParams()
   const botQuery = api.bot.getById.useQuery(id as string)
 
+  const iframeRef = useRef<HTMLIFrameElement>(null)
+
   return (
     <>
       <SeoHead title="Bot" />
       <PageHeader className="border-b border-divider">
         <PageHeaderTitle>Preview</PageHeaderTitle>
       </PageHeader>
       <PageContent className="bg-gray-50">
-        <div className="border rounded-lg min-h-[calc(100dvh-240px)] bg-white relative">
+        <Button
+          type="button"
+          onClick={() => {
+            iframeRef.current?.contentWindow?.postMessage(
+              JSON.stringify({ type: 'test' }),
+              '*',
+            )
+          }}
+        >
+          Send test message to widget iframe
+        </Button>
+        <iframe
+          ref={iframeRef}
+          src="http://localhost:3001"
```

Comment:
This should be configurable, right ?
Default value should be url of dev env instead ? Would be nice if we have something working out of the box.